mpris2controller

```diff
@@ -226,7 +252,7 @@ def main():
             description='A small daemon for the Linux desktop that '
                         'intelligently controls MPRIS2-compatible media players')
     parser.add_argument('call', nargs='?', metavar='METHOD',
-            help='method to call on daemon (PlayPause, Next, Previous, or Quit)')
+            help='method to call on daemon (PlayPause, Next, Previous, Quit, Status, or Metadata)')
```

Comment:
Might want to add these to the list at the end of README as well :P

Reply:
Sure, I forgot about it :sweat_smile: